public/scripts/extensions/expressions/style.css

```diff
@@ -10,6 +10,49 @@
     width: 100vw;
 }
 
+body.hLetterBox #visual-novel-letterbox-one {
+    width: 100vw;
+    height: calc(0vh + var(--letterboxSize));
+    top: 0;
+    position: absolute;
+    background-color: var(--SmartThemeChatLetterboxColor);
+    z-index: 3;
```

Comment:
Is the z-index needed here, for all those blocks? Can you explain that a bit?
I tried toggling it off in dev view and it seemed to work.

Not the biggest fan of z-index if there are ways around it.

Reply:
Z-Index 3 is used, primarily to have the letterbox be over character sprites. Mostly a stylistic choice from me based off some animes that use it having the character underneath the bars themselves.